wandb/sdk/wandb_init.py

```diff
@@ -970,7 +970,7 @@ def _attach(
     if run is None:
         run = Run(settings=settings)
     else:
-        run._init()
+        run._init(settings=settings)
```

Comment:
why is this needed?

Reply:
because otherwise the run instance doesn't have a settings object when it is being attached